src/main/java/apihw/constants/RequiredParameters.java

```
@@ -0,0 +1,26 @@
+package apihw.constants;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class RequiredParameters {
+    public static Properties properties;
+
+    static {
+        try {
+            properties = new Properties();
+            properties.load(new FileInputStream("src/test/resources/test.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
```

if you don't read from property file, your tests don't make any sense, no request will be sent. You need to rethrow the exception here (and add the appropriate info message)